doc/radosgw/metrics.rst

```diff
@@ -117,7 +117,7 @@ To view op metrics in the Ceph Object Gateway go to the ``rgw_op`` sections of t
 Op Metrics Labels
 -----------------
 
-Op metrics can also be tracked per-user or per-bucket. These metrics are exported to Prometheus with labels like Bucket = {name} or User = {userid}::
+Op metrics can also be tracked per-user or per-bucket. These metrics are exported to Prometheus with labels like ``Bucket = {name}`` or ``User = {userid}``::
```

Comment:
The label names should be lowercase.  If I ever find an exporter returning mixed case labels I will track down the author and fart in their general direction.  

Suggest taking out “like”